packages/wonder-blocks-labeled-field/src/components/__tests__/labeled-field.test.tsx

```diff
@@ -136,86 +151,54 @@ describe("LabeledField", () => {
         expect(errorMessageEl).toBeInTheDocument();
     });
 
-    it("LabeledField adds testId to label", () => {
```

Comment:
I removed these tests because we have other tests in the `testId` section covering that the test id is set on the different elements!